### ISSUE TITLE
Properties are not cleared when using a new connector

### ISSUE THREAD

**Issue by jimmy-jia** [tableau/webdataconnector#104]
The information under the properties area of the simulator (Connection Name, Connection Data, etc) are not reset when you use a new connector. For instance, if you use the Earthquake Multitable connector followed by a connector that does not set either the Connection Name or Connection Data, the values from the previously simulated WDC still appear.

**Comment by jagreene** [comment]
I'm not sure the best part in the user interaction we should reset these values. @lbrendanl any ideas?

**Comment by lbrendanl** [comment]
I would say as soon as a new connector is launched (most likely in the interactive phase, but wouldn't have to be), we should clear the old connector properties.

**Comment by jagreene** [comment]
My one concern is that there isn't an easy way to tell whether or not the
user intentionally modified those properties in-between simulations and
then we would be overwritting their preferences. For example at the end of
the simulation a user changes the locales to de-DE and then starts a new
simulation, we would then default back to en-US.

**Comment by lbrendanl** [comment]
Totally agree there, I was suggesting we somehow detect that the simulation that was launched was to a new wdc (with a different URL).  I am comfortable won't fixing this though if the fix is hacky or a large investment; this is somewhat of an edge case (with an easy workaround).

**Comment by jagreene** [comment]
yeah it would be pretty easy to detect starting with a new url. Its just that if we reset then it'll its hard to know which of the properties that are there the user might have put intentionally before starting the simulation.

**Comment by lbrendanl** [comment]
Yeah that's a fair point, it'd be worth keeping that use case.  I am going to close this as a won't fix then.  Thanks!
